### ISSUE TITLE
Roku cast videos not working

### ISSUE THREAD

**Issue by mirroringcontact** [ConnectSDK/Connect-SDK-iOS#246]
Since October 31, I can't stream videos on Roku. I see just a black screen for 0.5 seconds and after that the media player closes

**Comment by mirroringcontact** [comment]
Yes. You are right. I fixed this problem. I create custom channel and it works good.

**Comment by trung10101995** [comment]
> Yes. You are right. I fixed this problem. I create custom channel and it works good.

Can you share it?

**Comment by siam009** [comment]
> Yes. You are right. I fixed this problem. I create custom channel and it works good.

Can you share it?

**Comment by mirroringcontact** [comment]
> > Yes. You are right. I fixed this problem. I create custom channel and it works good.
> 
> Can you share it?

default player is working now. 
image fix:
replace `NSString *applicationPath = [NSString stringWithFormat:@"15985?t=p&u=%@&tr=crossfade",` to
`NSString *applicationPath = [NSString stringWithFormat:@"15985?t=p&u=%@&h=%%20&k=%%20",` 
 
video fix:
 applicationPath = [NSString stringWithFormat:@"15985?t=v&u=%@&k=(null)&videoName=%@&videoFormat=%@"
 to 
  `applicationPath = [NSString stringWithFormat:@"15985?t=v&u=%@&h=%%20&k=%%20"`

**Comment by Gopal02024** [comment]
@dylancom ,
Roku device not connect.
Any solution?
